operator/docs/dev/security/results.golint.txt

```diff
@@ -1,36 +1,34 @@
-controllers/internal/csiscaleoperator/csiscaleoperator.go:125:1: exported method CSIScaleOperator.GetDefaultImage should have comment or be unexported
-controllers/internal/csiscaleoperator/csiscaleoperator.go:160:1: exported method CSIScaleOperator.GetKubeletRootDirPath should have comment or be unexported
-controllers/internal/csiscaleoperator/csiscaleoperator.go:171:1: exported method CSIScaleOperator.GetSocketPath should have comment or be unexported
-controllers/internal/csiscaleoperator/csiscaleoperator.go:179:1: exported method CSIScaleOperator.GetSocketDir should have comment or be unexported
-controllers/internal/csiscaleoperator/csiscaleoperator.go:187:1: exported method CSIScaleOperator.GetCSIEndpoint should have comment or be unexported
-controllers/internal/csiscaleoperator/csiscaleoperator_package.go:32:2: const snapshotStorageApiGroup should be snapshotStorageAPIGroup
-controllers/internal/csiscaleoperator/csiscaleoperator_package.go:33:2: const securityOpenshiftApiGroup should be securityOpenshiftAPIGroup
-controllers/internal/csiscaleoperator/csiscaleoperator_package.go:34:2: const storageApiGroup should be storageAPIGroup
-controllers/internal/csiscaleoperator/csiscaleoperator_package.go:35:2: const rbacAuthorizationApiGroup should be rbacAuthorizationAPIGroup
-controllers/internal/csiscaleoperator/csiscaleoperator_package.go:36:2: const coordinationApiGroup should be coordinationAPIGroup
-controllers/internal/csiscaleoperator/csiscaleoperator_package.go:37:2: const podSecurityPolicyApiGroup should be podSecurityPolicyAPIGroup
-controllers/internal/csiscaleoperator/csiscaleoperator_package.go:250:1: comment on exported method CSIScaleOperator.GenerateProvisionerClusterRoleBinding should be of the form "GenerateProvisionerClusterRoleBinding ..."
-controllers/syncer/csi_node.go:64:2: exported const EnvVarForCSINodeRegistrarImage should have comment (or a comment on this block) or be unexported
-controllers/syncer/csi_syncer.go:50:2: exported const EnvVarForCSIAttacherImage should have comment (or a comment on this block) or be unexported
+level=warning msg="[runner] The linter 'golint' is deprecated (since v1.41.0) due to: The repository of the linter has been archived by the owner.  Replaced by revive."
+api/v1/csiscaleoperator_types.go:316:2: struct field `Id` should be `ID` (golint)
+	Id string `json:"id"` // TODO: Rename to ID or id
+	^
+api/v1/csiscaleoperator_types.go:324:2: struct field `RestApi` should be `RestAPI` (golint)
+	RestApi []RestApi `json:"restApi"` // TODO: Rename to RESTApi or restApi
+	^
+api/v1/csiscaleoperator_types.go:354:6: type `RestApi` should be `RestAPI` (golint)
+type RestApi struct {
+     ^
+controllers/csiscaleoperator_controller.go:266:9: if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary) (golint)
```

Comment:
Would we like to resolve this golint suggestion in our code ?

Reply:
I would suggest to raise another issue for tracking so that we can fix them soon.

Reply:
Sure, we would want to use a better linter than golint, as it is deprecated, so we need to first explore on the available linters, then run the scans and fix the findings.